install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -838,7 +838,6 @@ data:
 {{- else }}
   enable-well-known-identities: "false"
 {{- end }}
-  enable-remote-node-identity: {{ .Values.remoteNodeIdentity | quote }}
```

Comment:
Same as above: Consider moving this commit to the beginning of the git history. Using a flag which is removed a previous commit might break git bisect.